Rubberduck.Core/UI/Command/MenuItems/ParentMenus/ProjectWindowContextParentMenu.cs

```diff
@@ -16,6 +16,7 @@ public ProjectWindowContextParentMenu(IEnumerable<IMenuItem> items, int beforeIn
     public enum ProjectExplorerContextMenuItemDisplayOrder
```

Comment:
💡 For a future PR we might want to convert this into a static class and make use of `nameof()` operators to allow for compile-type references to validate that the display order is always valid. Could leave a `TODO` comment, maybe.

Reply:
@bclothier we've always felt something was off with these... but never came up with anything else - might be good to expand on that idea in a discussion/issue; a static class makes sense as the menu setup is really more data than code...

Would it make sense to go one step further and pull that data from some `.yaml` file that just lists menu item names and the indentation determines what goes where and some attributes and switches could toggle `BeginGroup`...